src/storage/mvcc/txn.rs

```diff
@@ -181,7 +186,11 @@ impl<'a> MvccTxn<'a> {
                               key,
                               self.start_ts,
                               commit_ts);
-                        Err(Error::TxnLockNotFound)
+                        Err(Error::TxnLockNotFound {
+                            start_ts: self.start_ts,
+                            commit_ts: commit_ts,
+                            key: key.encoded().to_owned(),
```

Comment:
Could we avoid [to_owned](https://doc.rust-lang.org/std/borrow/trait.ToOwned.html#tymethod.to_owned) here, since it's usually been implemented by `clone`?